settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = "Item Management Service"
```

Comment:
The settings.gradle file is primarily used when you have a multi-module project.  But it is always a way to define the project name.  A little odd you use this file to define the project name instead of from the build.gradle.  

If we don't define this setting then the root folder name is used as the project name.